client/src/pages/tasks/Show.js

```diff
@@ -293,6 +293,29 @@ const TaskShow = ({ pageDispatchers }) => {
                       }
                     />
                   )}
+                  {task.descendantTasks && task.descendantTasks.length > 0 && (
+                    <TaskParentTask
+                      dictProps={Settings.fields.task.childTask}
+                      name="childTask"
+                      component={FieldHelper.ReadonlyField}
+                      humanValue={
+                        task.descendantTasks && task.descendantTasks.length > 0 && (
+                          <>
+                            {task.descendantTasks.map((tasks) => {
+                              console.log(tasks)
+                              return (
+                                <tr>
```

Comment:
Is this inside a table? Maybe have a look at how childrenOrgs are shown in pages/organizations/Show.js and use the same layout.